setup.py

```diff
@@ -34,6 +34,7 @@
         'cloudify-common>=4.5.0',
         # libvirt-6.0 requires python3
         'libvirt-python>=4.5.0,<6.0',
-        "cloudify-utilities-plugins-sdk==0.0.17", # cdrom create code
+        # cdrom create code
+        "cloudify-utilities-plugins-sdk==0.0.20",
```

Comment:
I think there is even a _compat module in utilities SDK that you can use instead of writing one specifically for this.